owltorrent-core/src/main/java/edu/rice/owltorrent/core/serialization/SHA1Encryptor.java

```diff
@@ -0,0 +1,35 @@
+package edu.rice.owltorrent.core.serialization;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.Arrays;
+
+/** SHA1 Encryptor */
+public class SHA1Encryptor {
+
+  /**
+   * Encrypt the input string using SHA1 algorithm
+   *
+   * @return the raw 20-byte SHA1 hash
+   */
+  public static byte[] encrypt(String input) {
```

Comment:
We most likely need a signature for byte[] too.

Reply:
I am not sure what I should add here. It will be a raw 20-byte array for hash. Could you elaborate on this?

Reply:
So basically have another signature "byte[] encrypt(byte[] input)", since we expect a lot of out data to be in bytes instead of as a string. And then this signature would be something like "return encrypt(str.getBytes());" instead.